### PR TITLE
Use path params instead of query params in details page tabs

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
## Fixes 
[HAC-2571](https://issues.redhat.com/browse/HAC-2571)

## Description
Use URL path params rather that search params for tabs in details pages.

## Type of change
- [x] Feature

### CODE REVIEW

File: src/components/Commits/AppRecentCommits.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ const AppRecentCommits = ({ applicationName }) => {
             component={(props) => (
               <Link
                 {...props}
-                to={`/stonesoup/applications/${applicationName}?activeTab=components&hacbs=true`}
+                to={`/stonesoup/applications/${applicationName}/components&hacbs=true`}
```
[the remark this answers is not in the document]
Comment (christianvogt) [reply src/components/Commits/AppRecentCommits.tsx:81]: I think we can just remove the `hacbs=true` parameter.

File: config/remotePlugin.js [hunk 2]
```diff
@@ -207,7 +207,7 @@ module.exports = {
       type: 'console.page/route',
       properties: {
         path: '/stonesoup/applications/:appName',
-        exact: true,
+        exact: false,
```
Comment (christianvogt) [inline config/remotePlugin.js:210]: Can use react-router to parse the tab:
```
path: '/stonesoup/applications/:appName/:tab?',
```

**Review by rottencandy — APPROVED** [review]
/lgtm

**Comment by rohitkrai03** [comment]
Hey, I've merged the PR that fixed the e2e tests - https://github.com/openshift/hac-dev/pull/305. So going forward it would be good to run the e2e tests locally and paste the screenshot here on the PR before merging. Can you do that?

**Comment by jeff-phillips-18** [comment]
I'm still trying to get the tests to run locally. Waiting for authorization to App Studio early access:

image

**Comment by jeff-phillips-18** [comment]
e2e tests:
```
====================================================================================================

  (Run Finished)


       Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  add-component-to-application.spec.t      01:26       14        8        -        6        - │
  │    s                                                                                           │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  create-application-from-public-git-      01:51        9        9        -        -        - │
  │    source.spec.ts                                                                              │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  create-application-from-sample.spec      01:18        3        3        -        -        - │
  │    .ts                                                                                         │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        04:36       26       20        -        6        -  
```

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Pull request merged**